lib/util/message.ts

```diff
@@ -0,0 +1,36 @@
+import { MessageElement } from 'npm:@slack/web-api';
+import { adjectives, colors, animals } from 'npm:unique-names-generator';
+import { uniqueNamesGenerator } from 'npm:unique-names-generator';
+import emoji from 'npm:node-emoji';
+import redis from '../redis.ts';
+
+export function isUnsuitableMessage(message: MessageElement) {
+  return (
+    message.type !== 'message' || message.subtype !== undefined || !message.text
+  );
+}
```

Comment:
Slack does not export the `MessageElement` type, so for now I believe it's best to just keep these checks in the main file.
Especially since to keep full type-safety, we'd need to cast the type of message in the return type, were again we don't have the original type.

For example:
```ts
export const isSuitableMessage = (
  message: MessageElement
): message is MessageElement & {
  type: 'message';
  subtype: undefined;
  text: string;
} =>
  message.type === 'message' && message.subtype === undefined && message.text;
```